### PR TITLE
Add us.csv from NYT

### PR DESCRIPTION
**Pull request by TomGoBravo**
This PR addresses https://trello.com/c/DCwiX3xR/1301-ingest-usa-level-case-death-data-from-nyt

* Also force common_df.read_csv set_index to be keyword param
* fix warning about missing county in states data.

## Tested

`python ./scripts/update_nytimes_data.py` and `git difftool -t vimdiff data/cases-nytimes/timeseries-common.csv` shows only the new US data getting added.

Running `./run.py data update` in the covid-data-model repo reproduced identical data; the us data isn't used without further changes.

### CODE REVIEW

File: scripts/update_nytimes_data.py [hunk 1]
```diff
@@ -249,11 +249,22 @@ class Fields(GetByValueMixin, FieldNameAndCommonField, enum.Enum):
     DEATHS = "deaths", CommonFields.DEATHS
 
 
+# Similar to Fields, but without 'COUNTY'
```
Comment (mikelehen) [inline scripts/update_nytimes_data.py:252]: Fields=>CountyFields ?

**Review by mikelehen — APPROVED** [review]
🎉

File: scripts/update_nytimes_data.py (hunk 1 above)
Comment (TomGoBravo) [reply scripts/update_nytimes_data.py]: Fixed in 45fbbf1

**Comment by TomGoBravo** [comment]
[Ingest USA-level case/death data from NYT.](https://trello.com/c/DCwiX3xR/1301-ingest-usa-level-case-death-data-from-nyt)

**Pull request merged**